src/index.ts

```diff
@@ -329,7 +329,11 @@ export async function onFormSubmit(
       }
     }
 
-    let runBusinessLogicStatus:RunBusinessLogicStatus = {status: "running", logicResults: []};
+    let runBusinessLogicStatus:RunBusinessLogicStatus = {
+      status: "running",
+      logicResults: [],
+      logicDocsForQueuing: [],
```

Comment:
I think there's no need for this extra attribute since all logicResults  doc are for view logic queuing